frontend/src/views/HomeView.vue

```diff
@@ -1 +1,66 @@
-<template><h1>Test</h1></template>
+<template>
+  <div class="page-wrapper">
+    <div class="table-container">
+      <h2>Roles</h2>
+      <el-table :data="rolesStore.getRoles" style="width: 100%">
+        <el-table-column prop="name" label="Names" />
+        <el-table-column fixed="right" label="Actions" width="150">
+          <template #default="{ row }">
+            <el-button @click="handleDetail(row)" type="text" size="small">Detail</el-button>
+            <el-button @click="handleEdit(row)" type="text" size="small">Edit</el-button>
+          </template>
+        </el-table-column>
+      </el-table>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { onMounted } from 'vue'
+import { ElTable, ElTableColumn, ElButton } from 'element-plus'
+import { useRolesStore } from '@/stores/roles/index'
+
+const rolesStore = useRolesStore()
+
+onMounted(async () => {
+  await rolesStore.fetchRoles()
+})
+
+const handleDetail = (row: any) => {
+  console.log('Detail clicked for:', row)
+}
+
+const handleEdit = (row: any) => {
+  console.log('Edit clicked for:', row)
+}
+</script>
+
+<style scoped>
```

Comment:
```suggestion
<style scoped lang="scss">
```

Try to search what are the main features of SCSS and delight youself